tools/validators/instance_validator/tests/fake_telemetry/message_attributes_CHWS_WDT-17.json

```diff
@@ -3,6 +3,6 @@
   "deviceId": "CHWS_WDT-17",
   "deviceRegistryId": "US-SEA-BLDG1",
   "deviceRegistryLocation": "us-central1",
-  "deviceNumId": "123456",
+  "deviceNumId": "2619178366980753",
```

Comment:
change cloud_device_id to something more generic like 1234567890123456

Reply:
Done.